content/zh/_index.md

```diff
@@ -78,13 +77,13 @@ OpenTelemetry 是 100% 免费和开源的，
 
 {{% blocks/section color="secondary" type="cncf" %}}
 
-**OpenTelemetry 是一个 [CNCF][] [Incubating][] 项目**。<br> 这个项目是在合并
-OpenTracing 和 OpenCensus 项目之后成型的。
+**OpenTelemetry 是一个 [CNCF][] [孵化中][] 项目**。<br> 这个项目是由
+OpenTracing 和 OpenCensus 项目合并而诞生的。
 
 [![CNCF logo][]][cncf]
 
 [cncf]: https://cncf.io
 [cncf logo]: /img/logos/cncf-white.svg
-[incubating]: https://www.cncf.io/projects/
+[孵化中]: https://www.cncf.io/projects/
```

Comment:
```suggestion
[incubating]: https://www.cncf.io/projects/
```
Hugo does not have a good support for zh anchors, not sure this case ok or not. This change is optional.

Reply:
I believe this change is made so that "孵化中" can be properly linked, and it is connected with the review comment above.

By the way, isn't it a link instead of an anchor?

And preview is available here so that we could test them.

https://deploy-preview-4470--opentelemetry.netlify.app/zh/